_posts/2019-05-09-How-We-Release-Code.md

```diff
@@ -0,0 +1,96 @@
+---
+title: How we release open source projects
+author: Per Ploug
+date: 2019-05-09
+banner:
+  image: connected.jpg
```

Comment:
I couldn't find this image. Is it somewhere in the repo?

Reply:
yup, its in the banners folder - jekyll picks up automatically